vault/dendron.topic.publish.config.siteRootDir.md

```diff
@@ -0,0 +1,22 @@
+---
+id: h5afVt8ss9e4wXye78oUi
+title: siteRootDir
+desc: 'Set the directory where your website will be built.'
```

Comment:
Does this mean that publishing commands from the CLI will automatically be moving the built assets into `docs`, instead of users needing to do `mv .next/out docs`?

Reply:
Looks like the cli doesn't look at this config right now.

Reply:
k, I'll create a follow-up item on this